src/detect-engine-threshold.c

```diff
@@ -277,6 +342,7 @@ static int ThresholdHandlePacketSuppress(Packet *p,
             }
             break;
         case TRACK_RULE:
+        case TRACK_FLOW:
```

Comment:
What is this line doing ? And `ThresholdHandlePacketSuppress` ?

Reply:
it makes it explicit that TRACK_FLOW isn't supported for type suppress